### DISCUSSION TITLE
New Clan members not showing up as allies.

### DISCUSSION THREAD

**Discussion by alexkmartinez77** [codewars/codewars.com#2661] [Q&A]
We have a coding group using Codewars as a training platform. We cannot see new clan members and new clan members cannot see the rest of the clan. Any help would be greatly appreciated.

**Comment by soharaa** [comment]
How did you solve this?

**Comment by khanhtranngoccva** [comment]
There's an API that allows you to list how many members are in the clan, and exactly who as well. You can use it to create your own, objective leaderboard instead of having to rely on scraping the oldest clan member's account.

Anyway, the original discussion is here 👉 https://github.com/codewars/codewars.com/discussions/2710

For now, the user only follows up to around 500 members in the clan though, and trying again won't work.
